pages/windows/move-item.md

```diff
@@ -0,0 +1,36 @@
+# Move-Item
+
+> Move or rename files and directories.
+> More information: <https://learn.microsoft.com/powershell/module/microsoft.powershell.management/move-item>.
+
+- Rename a file or directory when the target is not an existing directory:
+
+`Move-Item {{path\to\source}} {{path\to\target}}`
+
+- Move a file or directory into an existing directory:
+
+`Move-Item {{path\to\source}} {{path\to\existing_directory}}`
+
+- Rename or move file(s) with specific name (do not treat special characters inside strings):
+
+`Move-Item -LiteralPath "{{path\to\source}}" {{path\to\file_or_directory}}`
+
+- Move multiple files into an existing directory, keeping the filenames unchanged:
+
+`Move-Item {{path\to\source1 , path\to\source2 ...}} {{path\to\existing_directory}}`
```

Comment:
The commas are necessary because that's how to declare multiple values in PowerShell. Additionally, they are not written as `{{path\to\source_key1, path\to\source_key2 ...}}` instead of `{{path\to\source_key1 , path\to\source_key2 ...}}` to distinguish from other commands that do not require this comma.

![image](https://github.com/tldr-pages/tldr/assets/17312341/da09e273-7d12-42fe-9af9-b9b7b3ff934d)